Tokenization_Samples/README.md

```diff
@@ -1,55 +1,81 @@
+**tokengroup** : `Defines a group name space in the configuration database. The VTS GUI Administrator must create token groups that match the tokengroup parameter names used in the application code.`
+
+**tokentemplate** : `VTS GUI Administrator-defined name for a group of properties that define tokenization operation.`
 
-tokentemplate : VTS GUI Administrator-defined name for a group of properties that define tokenization operation.
+**data** : `The data string to tokenize. The argument considers case and is limited to 128K.`
 
-data : The data string to tokenize. The argument considers case and is limited to 128K.
+**github link** : https://github.com/thalescpl-io/CipherTrust_Application_Protection/blob/TOK-2779/Tokenization_Samples/tokenize.py
 
-Syntax-
-$ python vts_tokenize.py -d -u <username>:<password> -l  <vts URL> -p <data or plaintext string> -O <outputFile> -tg <tokengroup> -tt <tokentemplate>
+**Syntax** :
 
-Example-
-$ python vts_tokenize.py -d -u root:ca\$hcow -l  https://192.0.2.0/vts -p DataToTokenizeDetokenize -O token -tg FF1_Tok_Group -tt FF1_Tok_Template
+```
+$ python tokenize.py -d -u <username>:<password> -l <vts URL> -p <data or plaintext string> -O <outputFile> -tg <tokengroup> -tt <tokentemplate>
+```
 
-Output : 
+**Example** :
 
-Header: {'User-Agent': 'python-requests/2.23.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Type': 'application/json', 'Content-Length': '104', 'Authorization': 'Basic cm9vdDpjYSRoY293'}
+```
+$ python tokenize.py -d -u <username>:<password> -l https://192.0.2.0/vts -p DataToTokenizeDetokenize -O token -tg FF1_Tok_Group -tt FF1_Tok_Template
+```
+
+## Output
+
+```
+Header: {'User-Agent': 'python-requests/2.23.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '_/_', 'Connection': 'keep-alive', 'Content-Type': 'application/json', 'Content-Length': '104', 'Authorization': 'Basic cm9vdDpjYSRoY293'}
 URL: https://192.0.2.0/vts/rest/v2.0/tokenize
 Data: {"tokengroup": "FF1_Tok_Group", "tokentemplate": "FF1_Tok_Template", "data": "DataToTokenizeDetokenize"}
 
 EzchFFKH33EGhopWc|Bb|TV(
 
 Completed write text to file name token.
+```
+
+#
 
-===================================================
+## detokenize
 
-vts_detokenize
----------------------
 Detokenize a given token using the tokengroup and tokentemplate.
 
-tokengroup : Defines a group name space in the configuration database. The VTS GUI Administrator must create token groups that match the tokengroup parameter names used in the application code.
+**tokengroup** : `Defines a group name space in the configuration database. The VTS GUI Administrator must create token groups that match the tokengroup parameter names used in the application code.`
 
-tokentemplate : VTS GUI Administrator-defined name for a group of properties that define tokenization operation.
+**tokentemplate** : `VTS GUI Administrator-defined name for a group of properties that define tokenization operation.`
```

Comment:
Maybe rename "The VTS GUI Administrator" to "Tokenization Server Administrator" to avoid the VTS keyword.